docs/API.md

```diff
@@ -18,12 +18,16 @@ There are a number of methods that a DB storage backend should implement:
     * .createSessionToken(tokenId, sessionToken)
     * .updateSessionToken(tokenId, sessionToken)
     * .sessionToken(id)
+    * .sessionTokenVerified(tokenId)
```

Comment:
> `Verified` seems like an odd choice for a name

As per this and @rfk's suggestion, the methods are now called `...TokenWithVerificationStatus`. Note that this isn't quite synonymous with the `sessionWithDevice` method, which drops the `Token` part. I would have done that here but it looks weird in the `keyFetchWithVerificationStatus` case.

> For example, it's not obvious to me when we'd ever call `keyFetchToken` in preference to `keyFetchTokenVerified`

It's a good point. Maybe for the pre-verified accounts (not sure if they request keys or not)? But either way, having both in the short term at least makes the deployment more straightforward.